src/main/java/com/griefcraft/modules/pluginsupport/Factions.java

```diff
@@ -99,4 +104,19 @@ public boolean canProtectLegacy(FPlayer fPlayer, Faction faction) throws Illegal
 
         return false;
     }
+
+    public void onMatchingFaction(LWCFactionMatcherEvent event) {
```

Comment:
Replace with corresponding event, checking for permissions (e.g. LWCAccessEvent).